_testdata/17/foo.go

```diff
@@ -1,8 +1,5 @@
-import "time"
-
 func seventeen() {
 	x := []int{7, 8}
 	y := (*[2]int)(x)
-	time.Now().GoString()
```

Comment:
The other code in this file already forces Mingo to compute version 1.17. Adding this call to `time.Now().GoString()` makes no difference, whether the relevant logic works or doesn't work.

The right way to test this is to put it in its own file (which I've done with `bar.go`). The presence of `time.Now().GoString()` should force Mingo to compute version 17 all by itself. I've confirmed that with your fix in place, it does, and without your fix, it doesn't.